controllers/dbaas.redhat.com/configmap.go

```diff
@@ -0,0 +1,101 @@
+  mediatype: image/svg+xml`
+
+	// controller/cache will not be ready during operator 'setup', use manager client & API Reader instead
```

Comment:
Since we're contributing this back, maybe a little more detail here. something like "method invoked from main.go prior to completion of manager start-up so controller/cache not yet ready. Use manager client & API Reader instead."